flexmeasures/data/models/forecasting/model_spec_factory.py

```diff
@@ -27,6 +41,81 @@
 """
 
 
+logger = logging.getLogger(__name__)
+
+
+class TBSeriesSpecs(SeriesSpecs):
+
+    """Define how to collect timely beliefs.
+    This sets up a class which to call `collect` on, together with parameters to pass.
+    The collect function should return a BeliefsDataFrame.
+    """
+
+    db: Engine
+    generic_asset_value_class: Any  # with collect method
+    collect_params: dict
+
+    def __init__(
+        self,
+        db_engine: Engine,
```

Comment:
I fail to remember why this parameter (`db_engine`) is necessary. I see you are using it just as 'timetomodel.speccing.DBSeriesSpecs' is using it, but how is it relevant? It's not being used here, nor in that class. 

I can't even find another mention of it anywhere else in timetomodel!

Can we do without it?

Reply:
I believe we can.